packages/cbioportal-frontend-commons/src/lib/apiClientCache.ts

```diff
@@ -142,13 +176,23 @@ export function cachePostMethodsOnClient(
             !excluded.includes(methodName)
     );
 
-    postMethods.forEach(methodName =>
+    postMethods.forEach(methodName => {
         cachePostMethod(
             obj.prototype,
             methodName,
             apiCacheLimit,
             log,
             sentryLog
-        )
-    );
+        );
+
+        if (window.location.hostname.includes('genie-private')) {
```

Comment:
we can expand this as necessary.  obviously is this temporary solution

Reply:
can we include `genie.cbioportal.org` as well? For `genie_public`